ecms_base/features/custom/ecms_person/config/install/field.storage.media.field_personal_photo_image.yml

```diff
@@ -5,8 +5,8 @@ dependencies:
     - file
     - image
     - media
-id: media.field_media_image
-field_name: field_media_image
+id: media.field_personal_photo_image
+field_name: field_personal_photo_image
```

Comment:
Is this going to work for existing "person" nodes?

Reply:
Photo has been a disabled field up until now

Reply:
Got it